keel-core/src/main/kotlin/com/netflix/spinnaker/keel/services/ApplicationService.kt

```diff
@@ -126,12 +133,38 @@ class ApplicationService(
   fun getResourceSummariesFor(application: String): List<ResourceSummary> {
     return try {
       val deliveryConfig = repository.getDeliveryConfigForApplication(application)
-      resourceStatusService.getResourceSummariesFor(deliveryConfig)
+      return deliveryConfig.resources.map { resource ->
+        resource.toResourceSummary(deliveryConfig)
+      }
     } catch (e: NoSuchDeliveryConfigException) {
       emptyList()
     }
   }
 
+  private fun Resource<*>.toResourceSummary(deliveryConfig: DeliveryConfig) =
```

Comment:
@emjburns I managed to move this back here and out of the (now renamed) `ResourceHistoryService`.